### PR TITLE
Allow retrieval of lensing phi for the sim of interest

### PR DESCRIPTION
**Pull request by msyriac**
This is WIP. I have only implemented this for SOStandalonePrecomputedCMB so far. But early comments / review is appreciated.

### CODE REVIEW

File: mapsims/cmb.py [hunk 1]
```diff
@@ -121,6 +124,10 @@ def __init__(
 
     def simulate(self, ch, output_units="uK_CMB"):
         return self.signal(nu=ch.band, fwhm_arcmin=so_utils.get_beam(ch.telescope, ch.band), output_units=output_units)
+    def get_phi_alm(self):
+        # Return the lensing potential (phi) alms corresponding to this sim
```
Comment (zonca) [inline mapsims/cmb.py]: turn comment into docstring

File: mapsims/cmb.py [hunk 2]
```diff
@@ -121,6 +124,10 @@ def __init__(
...
+        import healpy as hp
```
Comment (zonca) [inline mapsims/cmb.py]: it is fine if you import `healpy` at module level

File: mapsims/cmb.py [hunk 3]
```diff
@@ -141,3 +148,12 @@ def _get_cmb_map_string(cmb_dir, iteration_num, cmb_set, lensed, aberrated):
         iteration_num,
     )
     return filename
+
+def _get_phi_map_string(cmb_dir, iteration_num):
+    # Implements the lensing potential alms file naming convention
+    # Ideally the same function should be used when saving sims
```
Comment (zonca) [inline mapsims/cmb.py]: turn comment into docstring

File: mapsims/cmb.py (hunk 1 above)
Comment (zonca) [inline mapsims/cmb.py]: also please mention this functionality in the docs

**Comment by zonca** [comment]
also please run `black` to fix formatting

**Comment by msyriac** [comment]
Done.

File: mapsims/cmb.py (hunk 1 above)
Comment (zonca) [reply mapsims/cmb.py]: fixed in ce97e77e51229ca77316fcec185e8d6c6c21f2cd

**Pull request merged**